fe1-web/network/MessageApi.ts

```diff
@@ -243,7 +243,7 @@ export function requestCreateElection(
   const message = new SetupElection({
     lao: currentLao.id,
     id: Hash.fromStringArray(
-      EventTags.ELECTION, currentLao.id.toString(), currentLao.creation.toString(), name,
+      EventTags.ELECTION, currentLao.id.toString(), time.toString(), name,
```

Comment:
Was that a confusion between current Lao time and election creation time ?

Would it be possible to catch that in a test ?

Reply:
I am not very proficient with front-end tests. But it should be yes.